### PR TITLE
Encrypt instance parameters

### PR DESCRIPTION
**Pull request by polskikiel**
**Description**

Changes proposed in this pull request:

- Encrypted instance parameters with migration

**Related issue(s)**

### CODE REVIEW

File: components/kyma-environment-broker/internal/migrations/instance_parameters_test.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	"testing"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/migrations"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/storage"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInstanceParametersMigration_Migrate(t *testing.T) {
```
Comment (kjaksik) [inline components/kyma-environment-broker/internal/migrations/instance_parameters_test.go:13]: What do you think about adding a check after this migration that would check if the auth is encrypted?

Comment (polskikiel) [reply components/kyma-environment-broker/internal/migrations/instance_parameters_test.go:17]: ok so i have changed the test to use postgres storage and added legacy methods that allows me to write a properly working migration along with the unit test

**Review by kjaksik — APPROVED** [review]

**Pull request merged**